Sources/MosaiqueAssetsPicker/ViewControllers/MosaiqueAssetPickerViewController.swift

```diff
@@ -17,7 +17,7 @@ public protocol MosaiqueAssetPickerDelegate: class {
     /// - Wait for the images to be ready (will be provided with by the `didPickImages`
     func photoPicker(_ controller: UIViewController, didPickAssets assets: [AssetFuture])
 
-    func photoPickerDidCancel(_ pickerController: MosaiqueAssetPickerViewController)
+    func photoPickerDidCancel(_ pickerController: UIViewController)
```

Comment:
I think we should rename `pickerController` `controller ` to remain consistent with the other calls.

Reply:
Agree! Changing now…

Reply:
done!